### DISCUSSION TITLE
Recover an admin password without recovery key?

### DISCUSSION THREAD

**Discussion by tasagore** [globaleaks/globaleaks-whistleblowing-software#4351] [Q&A]
Hi

I have a small instance for testing and I can't login with my admin password. If I try to reset the password I receive an email with the reset link but it asks for the recovery key and I can't find it. I have access to the sqlite db and I can see the tree users I created months ago there but since only one of them can admin the instance I can't modify anything.

Can I reset the password in anyway or override the recovery key and reset the password using an email link? 

I don't have too much data inside since it's a testing enviroment but I have some templates that I don't want to loose.

Thanks in advance

**Comment by evilaliv3** [comment]
@tasagore : If the setup is made for testing purposes only if you want to pass it to me i migh reset the user completely and set a new password. (it is not a recovery of the user account as no one could do it)

Just pass me the db on community.globaleaks.org
